coralproject/talk
# [CORL-174] premod user

Still needs copy review. 

## What this does
- add "always premoderate user" as an option in the user status dropdown
- when set to true, comments by this user will be flagged as PREMOD and will need to be approved
- includes confirmation modal, filtering by premod status, and account history entires

## How to test
- select "always premoderate user" from the status dropdown for a user
- log in as the premoderated user, attempt to comment on a stream _that isn't already set to premod_
- the comment should be held for moderation
